crates/rolldown/tests/esbuild/dce/dce_of_using_declarations/artifacts.snap

```diff
@@ -17,8 +18,9 @@ using dispose_keep = { [Symbol.dispose]() {
 await using await_asyncDispose_keep = { [Symbol.asyncDispose]() {
 	console.log("side effect");
 } };
-using undef_keep = undefined;
-await using await_undef_keep = undefined;
+using undef_remove = void 0;
```

Comment:
no side effect detection missed `void 0` for all cases below.

Reply:
Yeah,  I have already created an issue to track it, let's merge this pr if other ci passed.